multi-progress.js

```diff
@@ -19,82 +21,75 @@ const mockInstance = {
   isTTY: false,
 };
 
-module.exports = (Progress) => {
-  class MultiProgress {
-    constructor(stream) {
-      this.stream = stream || process.stderr;
-      this.isTTY = this.stream.isTTY;
+module.exports = class MultiProgress {
+  constructor(stream) {
+    this.stream = stream || process.stderr;
+    this.isTTY = this.stream.isTTY;
 
-      if (!this.isTTY) {
-        return mockInstance;
-      }
-      
-      this.cursor = 0;
-      this.bars = [];
-      this.terminates = 0;
-    
-      return this;
+    if (!this.isTTY) {
+      return mockInstance;
     }
+    
+    this.cursor = 0;
+    this.bars = [];
+    this.terminates = 0;
+  
+    return this;
+  }
 
-    newBar(schema, options) {
-      options.stream = this.stream;
-      var bar = new Progress(schema, options);
-      this.bars.push(bar);
-      var index = this.bars.length - 1;
-
```

Comment:
Can you also replace newlines on L44, L49, and L67?

Reply:
done